<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UserOptions</class>
 <widget class="QWidget" name="UserOptions">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>191</width>
    <height>454</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>191</width>
     <height>451</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="currentIndex">
    <number>3</number>
   </property>
   <widget class="QWidget" name="category">
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <widget class="QPushButton" name="camaraBtn">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>40</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>camara</string>
     </property>
    </widget>
    <widget class="QPushButton" name="generalButton">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>100</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>general</string>
     </property>
    </widget>
    <widget class="QPushButton" name="polylineButton">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>170</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>polyline</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="camaraoptions">
    <widget class="QPushButton" name="backButton">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>30</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>back</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>70</y>
       <width>141</width>
       <height>121</height>
      </rect>
     </property>
     <property name="title">
      <string>move</string>
     </property>
     <widget class="QPushButton" name="rightButton">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>50</y>
        <width>41</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>right</string>
      </property>
     </widget>
     <widget class="QPushButton" name="leftButton">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>41</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>left</string>
      </property>
     </widget>
     <widget class="QPushButton" name="downButton">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>80</y>
        <width>41</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>down</string>
      </property>
     </widget>
     <widget class="QPushButton" name="upButton">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>20</y>
        <width>41</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>up</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>210</y>
       <width>141</width>
       <height>81</height>
      </rect>
     </property>
     <property name="title">
      <string>zoom</string>
     </property>
     <widget class="QPushButton" name="zoomInButton">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>20</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>zoom in</string>
      </property>
     </widget>
     <widget class="QPushButton" name="zoomOutButton">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>50</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>zoom out</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>320</y>
       <width>141</width>
       <height>101</height>
      </rect>
     </property>
     <property name="title">
      <string>titlt</string>
     </property>
     <widget class="QPushButton" name="titltUpButton">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>20</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>titlt up</string>
      </property>
     </widget>
     <widget class="QPushButton" name="titltDownButton">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>60</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>titlt down</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="generaloptions">
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>80</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>day/night</string>
     </property>
    </widget>
    <widget class="QPushButton" name="switchDayNightButton">
     <property name="geometry">
      <rect>
       <x>101</x>
       <y>81</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>night</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>120</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>compass</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>160</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>doppler</string>
     </property>
    </widget>
    <widget class="QPushButton" name="showCompassButton">
     <property name="geometry">
      <rect>
       <x>101</x>
       <y>120</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>show</string>
     </property>
    </widget>
    <widget class="QPushButton" name="showDopplerButton">
     <property name="geometry">
      <rect>
       <x>101</x>
       <y>160</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>show</string>
     </property>
    </widget>
    <widget class="QPushButton" name="backButton_2">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>30</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>back</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>200</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>traffic</string>
     </property>
    </widget>
    <widget class="QPushButton" name="showTrafficButton">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>200</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>show</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="polylineOptions">
    <widget class="QPushButton" name="backButton_3">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>40</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>back</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox_4">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>79</y>
       <width>151</width>
       <height>81</height>
      </rect>
     </property>
     <property name="title">
      <string>polyline</string>
     </property>
     <widget class="QPushButton" name="addPolylineButton">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>20</y>
        <width>91</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>add poly line</string>
      </property>
     </widget>
     <widget class="QPushButton" name="clearPolylineButton">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>50</y>
        <width>91</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>clear all polyline</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_5">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>190</y>
       <width>151</width>
       <height>111</height>
      </rect>
     </property>
     <property name="title">
      <string>circle</string>
     </property>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>46</width>
        <height>13</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="text">
       <string>radius</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="textEdit">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>20</y>
        <width>91</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>46</width>
        <height>13</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="text">
       <string>fill color</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>80</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="text">
       <string>stroke color</string>
      </property>
     </widget>
     <widget class="QPushButton" name="circleFillColorButton">
      <property name="geometry">
       <rect>
        <x>74</x>
        <y>50</y>
        <width>71</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="circleStrokeColorButton">
      <property name="geometry">
       <rect>
        <x>74</x>
        <y>80</y>
        <width>71</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>170</y>
       <width>171</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>click on the map to put circles</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
